src/lib/react-toast-notification/src/ToastController.tsx

```diff
@@ -42,10 +42,22 @@ class Timer {
 }
 
 export class ToastController extends Component<Props, State> {
-  timeout: typeof TimerType;
+  timeout: typeof TimerType = {
+    clear: () => {
+      // noop
+    },
+    pause: () => {
+      // noop
+    },
+    resume: () => {
+      // noop
+    },
+  };
```

Comment:
```suggestion
  timeout: typeof TimerType | null = null;
```

こうの方が読みやすいかも？(原作者の意図としては初期値はundefinedっぽそうだけど

Reply:
意図が undefined だったらオプショナルにしちゃった方がいいのかな

Reply:
```ts
timeout?: typeof TimerType;
```

Reply:
あーそれでもよさそうね

Reply:
fixed https://github.com/voyagegroup/ingred-ui/pull/1043/commits/f323c3e007b58d23b9771081e8bd23c516ae5095